packages/ui/components/forms/Checkbox/Checkbox.en.mdx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react'
+import Content from '../../Content'
+import Br from '../../Br'
+import Checkbox from '../Checkbox'
+import { Sandbox } from '@startupjs/docs'
+
+# Checkbox
+
+Checkbox allows user to select one or more items.
+
+## Basic Example
+
+```jsx
+import { Checkbox, Content, Br } from '@startupjs/ui'
+```
+
+```jsx example
+const [checked, setChecked] = useState(false)
+return (
+  <Content>
+    <Checkbox
+      label='Example Checbkbox'
+      value={checked}
+      onChange={(newState) => setChecked(newState)}
+    />
+    <Br />
+    <Checkbox
+      label='Disabled Checbkbox'
+      value={true}
+    />
+  </Content>
+)
+```
+
+## Variant Example.
+
+Checkbox can have `switch` or `checkbox` variants. Default variant is `checkbox`.
+
+```jsx
+import { Checkbox, Content, Br } from '@startupjs/ui'
+```
+
+```jsx example
+const [checked, setChecked] = useState(false)
+const [switched, setSwitch] = useState(false)
+return (
+  <Content>
+    <Checkbox
+      label='Example Checbkbox'
+      value={checked}
+      onChange={(newState) => setChecked(newState)}
+    />
+    <Br />
+    <Checkbox
+      variant='switch'
+      label='Example Switch'
+      value={switched}
+      onChange={(newState) => setSwitch(newState)}
+    />
+  </Content>
+)
+```
+
+## Checkbox Layout
+
+Checkbox can have `pure` or `rows` layout. Default layout is `rows`.
+In pure layout additional props `onFocus` & `onBlur` are available.
```

Comment:
It's probably a bug in our API that `onFocus` & `onBlur` are only available in `pure` layout. It should probably be available in any layout.

Reply:
They're actually available in both the layouts. I misunderstood.